### PR TITLE
Run DataStore specific integration tests

### PR DESCRIPTION
**Pull request by wcekan**
The integration test refactoring skipped all DataStore specific tests.

### CODE REVIEW

**Comment by clayreimann** [comment]
For some reason it looks like a lot more `Filtered` objects are being created now...

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by wcekan** [comment]
@clayreimann Seems Hibernate 5 uses an odd mechanism to generate IDs.   Each object of subclass BaseID gets a unique ID regardless of Entity table.

File: elide-integration-tests/src/test/java/com/yahoo/elide/tests/DataStoreIT.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2015, Yahoo Inc.
```
Comment (DeathByTape) [inline elide-integration-tests/src/test/java/com/yahoo/elide/tests/DataStoreIT.java]: Should be 2015

**Comment by DeathByTape** [comment]
:+1:

**Pull request merged**